src/glyph.rs

```diff
@@ -0,0 +1,141 @@
+use glib::translate::*;
+use pango_sys;
+use GlyphItem;
+use Item;
+use GlyphString;
+
+impl GlyphString {
+
+    pub fn num_glyphs(&self) -> i32 {
+        unsafe { (*self.to_glib_none().0).num_glyphs }
+    }
+
+    pub fn glyph_info(&self) -> Vec<GlyphInfo> {
+
+        let num_glyphs = self.num_glyphs();
+        let mut ret = Vec::new();
+
+        unsafe {
+            let glyphs: *mut pango_sys::PangoGlyphInfo = (*self.to_glib_none().0).glyphs;
+            if num_glyphs > 0 {
+                for x in 0..num_glyphs {
+                    ret.push(from_glib_none(
+                        glyphs.offset(x as isize),
```

Comment:
```suggestion
                        glyphs.add(x),
```

Reply:
Maybe this can be implemented as trait FromGlibContainerXXX on GlyphInfo?

Reply:
You mean so that this function is just calling `FromGlibContainerNone::from_glib_none_num(glyphs, num_glyphs)`?

Reply:
Yes

Reply:
Moved to a `FromGlibContainer` trait implementation